dist/css/colette.css

```diff
@@ -784,35 +786,83 @@ a:focus {
   outline: thin dotted;
 }
 h1 {
-  font-family: 'Open Sans WebFont', Arial, 'Helvetica Neue', Helvetica, sans-serif;
+  font-family: Arial, 'Helvetica Neue', Helvetica, sans-serif;
```

Comment:
Is this OK if we have "Arial" in generated CSS ?

Reply:
Yes because line 794 and 798

Reply:
But my concern was about the visual result in fact. Arial is not really the rollback font or am I missing something here ? ^^' Why there is Arial ?